appengine/flexible/endpoints/clients/service_to_service_gae_default/main.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Example of calling a Google Cloud Endpoint API with a JWT signed by
+Google App Engine Default Service Account."""
+
+import base64
+import json
+import time
+import httplib
+import webapp2
```

Comment:
separate third party imports from standard library imports (`nox -s lint` will show you lint errors).